### ISSUE TITLE
GFF load fails when it sees lnc_RNA feature type

### ISSUE THREAD

**Issue by jhumann** [tripal/tripal#997]
### System information
Tripal Version: 7.x-3.2
Drupal Version: 7.67
PostgreSQL Version: 10.10
PHP Version: 7.0.33

### Issue description
I am loading a gff of annotated features from a genome and the gff loader job fails when it encounters the feature type of lnc_RNA, which is a SO type.  The lnc_RNA is related to a gene, like a mRNA is to a gene, in the GFF files.  I can post a snippet of the GFF if needed.

#### Error messages and screenshots

Cannot find feature term 'lnc_RNA' on line 1119 of the GFF file

**Comment by bradfordcondon** [comment]
hi @jhumann  my first hunch would be that you dont have this term in your database.  Can you find it when searching the terms on your site?

edit- i checked on my standard tripal site, i do have the term, so we'd have to see your GFF to understand hte problem more

It would be great if you could share a) the full error message and b) a snippet of the GFF (or a link to the full GFF)

**Comment by jhumann** [comment]
Okay, we checked and the SO term for lnc_RNA was not on the CV table.  So, I tried to update the CV terms using the OBO Vocabulary Loader.  I selected the "Sequence Ontology" option and clicked Update Ontology Details.  I immediately get a message that says the SO vocabulary has been updated (seems a bit too quick).  However, the term is still not in the database.  I checked the OBO file at the URL that the loader uses (http://purl.obolibrary.org/obo/so.obo) and the term exists there.  Either I am doing something wrong, or for some reason our SO terms are not updating.

Note:  This is a database site that we converted from T2 to T3, so maybe there is something from that causing problems?

Any suggestions?
